src/main/java/rx/Observable.java

```diff
@@ -9305,5 +9306,263 @@ public void call(Subscriber<? super T> observer) {
             });
         }
     }
+    
+    /**
+     * @warn javadoc
+     */
+    @Experimental
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public static <T, S> Observable<T> create(Action1<ProducerState<T, S>> onRequest) {
```

Comment:
Does this conflict with Groovy and Clojure because of type erasure? I'm concerned that 2 overloads that take functions will collide.